src/NewChain/AuxiliaryChain.ts

```diff
@@ -274,12 +274,24 @@ export default class AuxiliaryChain {
     node.startSealer(gasPrice, targetGasLimit, bootKeyFile);
     // The sealer runs locally on this machine and the port is published to the host from the
     // docker container.
-    this.logInfo('waiting 5 seconds for the sealer port to become available');
-    await AuxiliaryChain.sleep(5000);
     // Has to be RPC and not WS. WS connection was closed before deploying the Co-Gateway.
     // Reason unknown. Possibly due to the fact that according to `lsof` node keeps opening new
     // connections.
     this.web3 = new Web3(`http://127.0.0.1:${this.nodeDescription.rpcPort}`);
+    const maxRetries = 5;
```

Comment:
Why is this hard coded here?

Reply:
Do you mean it has to be moved after the class declaration ?

Reply:
Not sure 🤔 We could add these "magic numbers" as private constants to the beginning of the class. I don't think it makes sense to pass this in the constructor. At least not for now.

Reply:
Added as private member of the class.